partials/tabs/projects.html

```diff
@@ -26,8 +26,6 @@
                 <br> </div>
               <div ng-show="project.collaborating_projects.length>0" class="project-overview-element clickable card-action"> <span ng-click="search(project)" class="pr-element-detail chip" ng-repeat="project in project.collaborating_projects">{{ project }}</span>
                 <br> </div>
-              <div ng-show="project.status=='crowded'" class="red white-text flag card-action center"> <span class="status-title">HIGH INVOLVEMENT</span>
-                <br> <span class="tiny">If you haven't put work into this, look out for other projects for better chances!</span> </div>
```

Comment:
I agree, that's probably a better way to handle this. I'll do it that way instead and resubmit.